packages/eui/src/components/form/form_control_layout/form_control_layout.tsx

```diff
@@ -66,174 +67,134 @@ export type EuiFormControlLayoutProps = CommonProps &
     inputId?: string;
   };
 
-export class EuiFormControlLayout extends Component<EuiFormControlLayoutProps> {
-  static contextType = FormContext;
-
-  static defaultProps = {
-    iconsPosition: 'absolute',
-  };
-
-  render() {
-    const { defaultFullWidth } = this.context as FormContextValue;
-    const {
-      children,
+export const EuiFormControlLayout: FunctionComponent<
+  EuiFormControlLayoutProps
+> = (props) => {
+  const { defaultFullWidth } = useFormContext();
+  const {
+    inputId,
+    className,
+    children,
+    icon,
+    iconsPosition,
```

Comment:
Should we set the default `absolute` here too again?

```suggestion
iconsPosition = 'absolute',
```

Reply:
Super good question! I _think_ I left it out because `EuiFormControlLayoutIcons` already defines it as defaulting to `absolute` here:

https://github.com/elastic/eui/blob/c6473d555e5cc4a64db15a43b021b79c1614daa6/packages/eui/src/components/form/form_control_layout/form_control_layout_icons.tsx#L53-L55

And the [current docs prop table](https://eui.elastic.co/#/forms/form-controls#form-control-layout) doesn't appear to list `absolute` as the default prop in any case. But it probably should. 😕 Let's do it!